WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -497,9 +487,9 @@ function(task, colIdx) {
 	htmlArr[idx++] = DwtId.getListViewItemId(DwtId.WIDGET_ITEM_FIELD, this._view, task.id, ZmItem.F_ITEM_ROW_3PANE);
 	htmlArr[idx++] = "'>";
 
-    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx);
+    idx = this._getAbridgedCell(htmlArr, idx, task, ZmItem.F_SUBJECT, colIdx, '', '', 'TaskSubject');
```

Comment:
can we do this generically? like how classes for flag/tag/attachment are getting applied
check ZmListView.prototype._getCellClass